src/containers/SignUpScreen/index.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  ImageBackground,
+  ScrollView
+} from 'react-native';
+import { connect } from 'react-redux';
+import { KeyboardAwareView } from 'react-native-keyboard-aware-view';
+import { object, func } from 'prop-types';
+
+import SignUpForm from '../../components/SignUpForm';
+import { signUp } from '../../actions/userActions';
+import styles from './styles';
+
+class SignUpScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.signUp = this.signUp.bind(this);
+    this.navigateToLogin = this.navigateToLogin.bind(this);
+  }
+
+  signUp(userData) {
+    const jsUser = userData.toJS();
+    jsUser.gender = jsUser.gender.toLowerCase();
+    const user = {
+      user: jsUser
+    };
+    this.props.userSignUp(user);
```

Comment:
I would simplify this code by naming the `jsUser` const just `user` then here you can just do: `this.props.userSignUp({ user });`.